games-strategy/7kaa/7kaa-2.15.4p1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="Seven Kingdoms: Ancient Adversaries"
+DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy\
+	with the addition of trade, diplomacy, and espionage.\
+	\
```

Comment:
No tabs here (in front of the lines) and in the next, new line can be added by skipping backslashes, plus space at the end of the first line (check after a build how HaikuDepot shows it:
```diff
-DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy\
-	with the addition of trade, diplomacy, and espionage.\
-	\
+DESCRIPTION="Seven Kingdoms, designed by Trevor Chan, brings a unique blend of Real-Time Strategy \
+with the addition of trade, diplomacy, and espionage.
+
```